apps/financial-aid/backend/src/app/modules/staff/staff.service.ts

```diff
@@ -63,11 +69,43 @@ export class StaffService {
     return { numberOfAffectedRows, updatedStaff }
   }
 
+  private async sendEmail(
+    input: CreateStaffDto,
+    municipalityName: string,
+    user: Staff,
+  ) {
+    if (input.roles.includes(StaffRole.EMPLOYEE)) {
```

Comment:
Just employee? Should admin and super admin not get email? 📫

Reply:
Nopes they get another one, which is not designed yet